elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -370,6 +370,12 @@ def _verify_bug_status(self, bugs):
                 status = f"{bug.status}: {bug.resolution}"
             self._complain(f"Bug {bug.id} has status {status}")
 
+    def _verify_bug_summary(self, bugs):
```

Comment:
Can we have unit tests for these?

Reply:
it can be tested in test_verify_bugs_with_sweep_cli test case